READ_BEFORE_UPDATE.md

```diff
@@ -1,5 +1,38 @@
 # Update Notes
 
+## Migrating from privacyIDEA 3.9.2 to eduMFA
+
+> [!CAUTION]
+>  * Make a backup of your existing installation! This backup should include the config files, the encryption keys and
+     the database.
+>  * Ensure that you use the latest supported privacyIDEA version 3.9.2 and then upgrade to eduMFA
+> 
+> These migration steps are applicable for the versions 1.0.0 to 1.5.1
```

Comment:
maybe we change this line until we release a version for which the instructions do NOT work
```suggestion
> These migration steps are applicable for all versions released up to now
```